tests/foreman/ui_airgun/test_domain.py

```diff
@@ -0,0 +1,249 @@
+# -*- encoding: utf-8 -*-
+"""Test class for Domain UI
+
+:Requirement: Domain
+
+:CaseAutomation: Automated
+
+:CaseLevel: Acceptance
+
+:CaseComponent: UI
+
+:TestType: Functional
+
+:CaseImportance: High
+
+:Upstream: No
+"""
+from fauxfactory import (
+    gen_string,
+    gen_utf8
+)
+import pytest
+
+from robottelo.constants import DOMAIN
+from robottelo.datafactory import parametrized
+from robottelo.decorators import (
+    parametrize,
+    tier2,
+    upgrade
+)
+
+
+def valid_domain_names(length):
+    """Returns pytest parametrize kwargs for valid domain names."""
+    max_len = 255 - len(DOMAIN % '')
+    if max_len - length < 0:
```

Comment:
Nitpick: maybe `if length > max_len`? :)

Reply:
yes, that's much better ;)